python/setup.py

```diff
@@ -69,10 +69,15 @@
 
 EXAMPLES_PATH = os.path.join(SPARK_HOME, "examples/src/main/python")
 SCRIPTS_PATH = os.path.join(SPARK_HOME, "bin")
+DATA_PATH = os.path.join(SPARK_HOME, "data")
 SCRIPTS_TARGET = os.path.join(TEMP_PATH, "bin")
 JARS_TARGET = os.path.join(TEMP_PATH, "jars")
 EXAMPLES_TARGET = os.path.join(TEMP_PATH, "examples")
+DATA_TARGET = os.path.join(TEMP_PATH, "data")
+LICENSES_PATH = os.path.join(SPARK_HOME, "licenses")
+LICENSES_TARGET = os.path.join(TEMP_PATH, "licenses")
 
+data_files = glob.glob(os.path.join(LICENSES_PATH, "*"))
```

Comment:
Oh no I mean, including the license files is reasonable its more is it necessary to use `data_files` here (and down bellow) when all of the other packages we are producing simply use `package_data` to specify the backing files.